components/Navbar.js

```diff
@@ -50,31 +54,71 @@ export default function Navbar() {
-        <div className="flex-shrink-0">
-          <Image className="block h-8 w-auto" src="/images/pantry_logo.png" alt="Pantry logo" priority="true" height="32" width="32"/>
-        </div>
-        <div className="">
-          <div className="flex">
+  function toggleShowTabs() {
+    setShowTabs(!showTabs)
+  }
+
+    return (
+    <nav className="bg-pantry-blue-500 text-white p-4 flex flex-wrap justify-between items-center pr-10 overflow-visible flex-shrink-0">
+      {/* Pantry Logo */}
+      <a className="" href="/">
+        <Image className="block h-10 w-auto" src="/images/pantry_logo.png" alt="Pantry logo" priority="true" height="32" width="32"/>
+      </a>
+
+      {/* User Info / Sign-In / Sign-Out */}
+      <div className='flex-grow md:order-2 md:flex-grow-0'>
+        {!user ? <a className="bg-gray-100 text-gray-700 rounded px-3 py-1 float-right" href="/signin">Sign In</a> :
+          <div className="flex flex-col float-right">
+            <button className="focus:outline-none w-8 h-8 rounded-full bg-gray-100 text-gray-700 font-semibold truncate" onClick={toggleUserDropdown}>
+              {name.split(" ").map((n)=>n[0]).join("") /* circle with initials */}
+            </button>
+            <div className={(showUserInfo ? "" : "hidden ") + 
+              "z-50 my-4 text-base list-none bg-white rounded divide-y divide-gray-200 shadow absolute mt-10 right-0 mr-5"} id="dropdown">
+              <div className="py-3 px-4">
+                <span className="block text-sm text-gray-900 font-medium">{name}</span>
+                <span className="block text-sm text-gray-500 truncate font-normal">{userType}</span>
+              </div>
+              <ul className="py-1" aria-labelledby="dropdown">
+              {userType == "Administrator" ? <li>
+                <a href="/admin" className="block py-2 px-4 text-sm text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">Admin Page</a>
+              </li> : ""}
+              <li>
+                <a href="/signout" className="block py-2 px-4 text-sm text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">Sign out</a>
+              </li>
+              </ul>
+            </div>
+          </div>
+        }
+      </div>
+
+      {/* Tabs */}
+      {/* drop-down menu button for small screens */}
+      <button type="button" onClick={toggleShowTabs}
+        className="inline-flex items-center p-1 ml-3 text-sm text-gray-500 rounded-lg md:hidden hover:bg-gray-100 focus:outline-none" aria-controls="mobile-menu-2" aria-expanded="false">
+        <svg className="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fillRule="evenodd" d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z" clipRule="evenodd"></path></svg>
```

Comment:
nit: moving these SVGs out into their own file so they can be used throughout the project